src/util/RegistrationFormFactory.js

```diff
@@ -113,16 +113,31 @@ const checkSubSchemas = function(fieldName, model, subSchemas, showError) {
     } else {
       ele.children('p').addClass('default-schema');
     }
+
+    // clear role="status" on <p>
+    ele.children('p').removeAttr('role');
+
+    // reset errors
     ele.removeClass('subschema-satisfied subschema-unsatisfied subschema-error');
+
+    // validate
     if (checkSubSchema(subSchema, value, model)) {
+      // passed
       ele.addClass('subschema-satisfied');
       ele.find('p span').removeClass('error error-16-small');
       ele.find('p span').addClass('confirm-16');
     } else {
+      // failed
       if (showError) {
         ele.find('p span').removeClass('confirm-16');
         ele.find('p span').addClass('error error-16-small');
         ele.addClass('subschema-error subschema-unsatisfied');
+
+        // set role="status" on <p>
+        ele.find('p').attr('role', 'status');
+        
+        // ele.find('p .sr-only').length === 0 && ele.find('p').prepend('<span class="sr-only" >Error</span>');
+        // ele.find('.sr-only').length === 0 && ele.prepend('<span class="sr-only">Error</span>');
```

Comment:
Flagging for removal if the commented code is unused in the final PR

Reply:
Those are the other options. B and C left them for now to easily try them out. I'll remove the code we don't use